### PR TITLE
Redo FK Validation fix

### PR DESCRIPTION
**Pull request by LordSimal**
See #841

### CODE REVIEW

File: src/Command/ModelCommand.php [hunk 1]
```diff
@@ -804,14 +811,14 @@ public function getValidation(Table $model, array $associations, Arguments $args
...
-     * @param array<string> $primaryKey The primary key field. Unused because PK validation is skipped
+     * @param bool $isForeignKey Indicates if the currently precessed field is a foreign key
      * @return array Array of validation for the field.
      */
     public function fieldValidation(
         TableSchemaInterface $schema,
         string $fieldName,
         array $metaData,
-        array $primaryKey
+        bool $isForeignKey = false
```
Comment (markstory) [inline src/Command/ModelCommand.php:821]: Won't this run into BC errors with userland extensions? Should we put the `isForeignKey` attribute into `metaData` instead?

Comment (ADmad) [reply src/Command/ModelCommand.php:821]: This is targeting `3.x`, so we can make breaking changes. But yes we should target `2.x` and put the `isForeignKey` in the `metaData`.

Comment (LordSimal) [reply src/Command/ModelCommand.php:821]: So #841 should be fine in this regard

**Comment by LordSimal** [comment]
Since we already put that info into the `$field` array in https://github.com/cakephp/bake/pull/841 I will close this

**Pull request closed without merging**